support-frontend/assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -123,4 +124,28 @@ export const tests: Tests = {
     seed: 6,
     targetPage: auOnlyLandingPage,
   },
+
+  removeDigiSubAddressTest: {
+    type: 'OTHER',
+    variants: [
+      {
+        id: 'control',
+      },
+      {
+        id: 'noAddress',
+      },
+    ],
+    audiences: {
+      ALL: {
+        offset: 0,
+        size: 1,
+      },
+    },
+    isActive: true,
+    referrerControlled: false,
+    seed: 7,
```

Comment:
unrelated, but `landingPageDesignSystemTest` and `fancyAddressTest` both have the same seed, I presume they weren't running at the same time